python/tk_multi_loader/utils.py

```diff
@@ -328,3 +328,33 @@ def resolve_filters(filters):
                 resolved_filter.append(field)
         resolved_filters.append(resolved_filter)
     return resolved_filters
+
+def get_thumbnail_field_for_item(item, use_version_thumbnail_as_fallback=True):
+    """
+    Get the field to use for the thumbnail for the given item.
+
+    Check if a thumbnail is available for the given item, and return
+    the field name to download it from. If not available and falling back
+    on the Version is allowed, check if one is available for it and,
+    if so, return the dotted field to use to retrieve it.
+
+
+    If there's no image field or no image to use, it returns None.
+
+    :param item: The QStandardItem to get the thumbnail field for.
+    :param use_version_thumbnail_as_fallback: Whether to use the Version's
+        thumbnail as a fallback if the item doesn't have one.
+    :returns: A SG field, e.g. "image", or ``None``.
+    """
+    sg_data = item.get_sg_data()
+    if not sg_data:
+        return None
+    # When it's an empty thumbnail, it's an AWS link with a "no_preview_t.jpg" image.
```

Comment:
Any easy way to check it is an AWS link?

Reply:
there's an easy way, yes. But if you upload a `no_preview_t.jpg` image, it'll still be an AWS URL. The only use case would be someone putting an external URL as a thumbnail, with `no_preview_t.jpg` in the URL...